idaes/models/properties/helmholtz/helmholtz.py

```diff
@@ -1,1793 +1 @@
-            pub_phlist, rule=rule_energy_density_terms
-        )
-
-    def get_material_flow_terms(self, p, j):
-        return self.material_flow_terms[p]
-
-    def get_enthalpy_flow_terms(self, p):
-        return self.enthalpy_flow_terms[p]
-
-    def get_material_density_terms(self, p, j):
-        if p == "Mix":
-            return self.dens_mol
-        else:
-            return self.dens_mol_phase[p]
-
-    def get_energy_density_terms(self, p):
-        return self.energy_density_terms[p]
-
-    def default_material_balance_type(self):
-        return MaterialBalanceType.componentTotal
-
-    def default_energy_balance_type(self):
-        return EnergyBalanceType.enthalpyTotal
-
-    def get_material_flow_basis(b):
-        return MaterialFlowBasis.molar
-
-    def define_state_vars(self):
-        return self._state_vars_dict
-
-    def define_display_vars(self):
-        return {
-            "Molar Flow (mol/s)": self.flow_mol,
-            "Mass Flow (kg/s)": self.flow_mass,
-            "T (K)": self.temperature,
-            "P (Pa)": self.pressure,
-            "Vapor Fraction": self.vapor_frac,
-            "Molar Enthalpy (J/mol)": self.enth_mol_phase,
-        }
-
-    def extensive_state_vars(self):
-        return self.extensive_set
-
-    def intensive_state_vars(self):
-        return self.intensive_set
-
-    def model_check(self):
-        pass
-
-    def calculate_scaling_factors(self):
-        super().calculate_scaling_factors()
-
-        sf_flow = iscale.get_scaling_factor(self.flow_mol, default=1)
-        sf_enth = iscale.get_scaling_factor(self.enth_mol, default=1)
-        sf_inte = iscale.get_scaling_factor(self.energy_internal_mol, default=1)
-        sf_dens = iscale.get_scaling_factor(self.dens_mol, default=1)
-        sf_pres = iscale.get_scaling_factor(self.pressure, default=1)
-        for v in self.material_flow_terms.values():
-            iscale.set_scaling_factor(v, sf_flow)
-        for v in self.enthalpy_flow_terms.values():
-            iscale.set_scaling_factor(v, sf_enth * sf_flow)
-        for k, v in self.energy_density_terms.items():
-            if k == "Mix":
-                iscale.set_scaling_factor(v, sf_inte * sf_dens)
-            else:
-                sf_inte_p = iscale.get_scaling_factor(
-                    self.energy_internal_mol_phase[k], default=1
-                )
-                sf_dens_p = iscale.get_scaling_factor(self.dens_mol_phase[k], default=1)
-                iscale.set_scaling_factor(v, sf_inte_p * sf_dens_p)
-        try:
-            iscale.set_scaling_factor(self.eq_sat, sf_pres / 1000.0)
-        except AttributeError:
-            pass  # may not have eq_sat, and that's ok
-        try:
-            iscale.set_scaling_factor(self.eq_complementarity, sf_pres / 10)
-        except AttributeError:
-            pass  # may not have eq_complementarity which is fine
+from idaes.models.properties.general_helmholtz import *
```

Comment:
Should we make the a deprecated path so that we can eventually remove this entirely?

Reply:
Yeah.  That's my plan.  I wan't sure what to do.  I don't want to update everything to use the new property package yet, so I didn't want to add bunch of new deprecation warnings.  There are too many moving parts here.  Maybe after this PR I can update our models and tests and add the deprecation warning.

Reply:
Lets wait until after the v2.0 release so that we can have one release with no deprecations.